Domonic/Domonic/Views/Movies/ToRent.cshtml

```diff
@@ -8,22 +8,22 @@
     returning a results page displaying: movies rented, date rented, return date, and thank you message
 *@
 <div class="container">
-    @using (Html.BeginForm("Rented", "Movies"))
+    @using (Html.BeginForm("ToRent", "Movies"))
     {
         foreach (var item in Model.AvailableMovies)
         {
             <div class="checkbox">
                 <label>
                     <input type="checkbox"
-                           name="All Movies"
+                           name="SelectedMovies"
```

Comment:
Domonic, this was also very important.  The checkbox name has to match the variable name of your IList<string> of selected items (i.e., "SelectedMovies")